app/src/main/java/net/programmierecke/radiodroid2/FragmentPlayerFull.java

```diff
@@ -485,6 +487,15 @@ private void fullUpdate() {
 
         timedUpdateTask.run();
 
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getContext());
+        final boolean show_album_art = prefs.getBoolean("settings_show_album_art", true);
+        if (show_album_art) {
+            pagerArtAndInfo.setVisibility(View.VISIBLE);
+        }else{
+            pagerArtAndInfo.setVisibility(View.GONE);
```

Comment:
The pager has two pages, first is album art, second is info about station and any other misc info we would want to show. So instead of hiding it we can show the second page of the pager. Also we want the same if `LastFMAPIKey` is empty.

Reply:
1. i thought really hiding is what was meant by the request. for me it was fine before.
2. do we use lastfm somewhere? what do you mean with lastfmapikey? ?????????

Reply:
> i thought really hiding is what was meant by the request.

Ok, I think we now both got it wrong. The plan is to not even request the album art and then not show the page in pager of it.

>  do we use lastfm somewhere? what do you mean with lastfmapikey? ?????????

I think you missed this part =) Yeah there is not many provider for album art, like there is lastfm, apple something, and some very small providers that lack a lot of data. So currently the album art is fetched via lastfm. I emailed you about this, but it's probably got lost.

Reply:
but the last thing i remember was that we can't use last fm because we cannot distribute a key?

Reply:
I've sent you email to your web.de address.

Reply:
this gets too complicated, i remove this enhancement from this release

Reply:
then you'd just need to remove first page of the pager, since there is no lastfm apikey - nothing will be requested

Reply:
at the moment it displays the stream icon, don't you like this?

Reply:
Personally I think it's redundant, I thought to add visualizer like https://github.com/gauravk95/audio-visualizer-android but I don't know. However I don't have a strong opinion here. You can leave it as it is.

Reply:
visualizer seems cool, but i don't want to add more features for 0.80, i want to push it to beta soon. could you please check changelog.md file if you want to add anything?

Reply:
ok, I'll check changelog later today or tomorrow.

Reply:
i am scrolling through last fm terms: if we ever want to consider adding lastfm, we need to at least mention them. so we have to add a licences page of some sort that mentions we use their api.
`You must credit Last.fm and include links to the Last.fm site when You use the Last.fm Data.`